spacewalk/config/etc/httpd/conf.d/zz-spacewalk-www.conf

```diff
@@ -65,11 +65,13 @@ ProxyTimeout 600
 </Proxy>
 RewriteRule ^/rhn/Login2\.do ajp://localhost:8009/rhn/manager/login [P]
 RewriteCond %{REQUEST_URI} !^/rhn/websocket
+RewriteCond %{REQUEST_URI} !^/rhn/websockify
 RewriteRule ^/rhn(.*) ajp://localhost:8009/rhn$1 [P]
 </IfModule>
 
 <IfModule proxy_wstunnel_module>
 ProxyPass "/rhn/websocket" "ws://localhost:8080/rhn/websocket"
+ProxyPass "/rhn/websockify" "wss://localhost:8050/"
```

Comment:
Just curious, why it is passed to secure ws. Why is not apache used terminate the tls?

Reply:
Otherwise I'm getting problems with the secure connection to spice and vnc